### ISSUE TITLE
Text colors are not accessible by visually impaired people, meet WCAG

### ISSUE THREAD

**Issue by alexgleason** [Uptwinkles/uptwinkles#18]
Web Content Accessibility Guidelines (WCAG) define a certain level of color contrast text must have in order to appear legible to people who are visually impaired. Our current body color `#3FC3B6` only has a contrast ratio of 2.17:1 over white, and therefore fails both levels of the guidelines.

![screenshot from 2017-05-11 10 21 03](https://cloud.githubusercontent.com/assets/3639540/25954250/a71022b0-3633-11e7-93e6-c738359dc55f.png)

I would like to meet the AAA guidelines, but it will likely require some sacrifice of aesthetics. In my opinion, especially for a site like Uptwinkles, it's worth sacrificing some visual appeal to be more accessible. Usually AA can be achieved without much sacrifice.

Here's one example color checker of many: http://webaim.org/resources/contrastchecker/

**Comment by alexgleason** [comment]
The latest deploy has the old colors again.
